.gitlab-ci.yml

```diff
@@ -0,0 +1,15 @@
+before_script:
+  - apt-get update
+stages:
+  - test
+  
+tcpecho_test:
+  image: ubuntu:18.04
+  stage: test
+  script:
+  # Installing mono
+    -  chmod 755 ./install_mono_from_microsoft_deb_packages.sh
+    -  chmod 755 ./make.sh  
```

Comment:
please remove the chmods, they are not needed if you commit the files with ugo+x

Reply:
The Windows Github client does not support changing the files permissions (or maybe I am missing something?). Anyway I pushed from Linux so it should be ok.